media/CfgEditor/displaycfg.js

```diff
@@ -0,0 +1,153 @@
+ */
+
+import oneOptimizationList from './one-optimizations.json' assert {type : 'json'};
+import {updateImportUI} from './updateUI.js';
+
+export function displayCfgToEditor(cfg) {
+  const checkboxImport = document.getElementById('checkboxImport');
+  const checkboxOptimize = document.getElementById('checkboxOptimize');
+  const checkboxQuantize = document.getElementById('checkboxQuantize');
+  const checkboxCodegen = document.getElementById('checkboxCodegen');
+  const checkboxProfile = document.getElementById('checkboxProfile');
+
+  checkboxImport.checked = false;
+  checkboxOptimize.checked = false;
+  checkboxQuantize.checked = false;
+  checkboxCodegen.checked = false;
+  checkboxProfile.checked = false;
+
+  // 'one-build' is replaced to 'onecc' when loaded
+  const onecc = cfg['onecc'];
+  if (!onecc) {
+    return;
+  }
+
+  if (onecc['one-import-tf'] === 'True') {
+    checkboxImport.checked = true;
+    const oneImportTF = cfg['one-import-tf'];
+    const modelFormat = oneImportTF ?.['model_format'];
+    if (modelFormat === undefined || modelFormat === 'graph_def') {
+      document.getElementById('importInputModelType').value = 'pb';
+    } else if (modelFormat === 'saved_model') {
+      document.getElementById('importInputModelType').value = 'saved';
+    } else {
+      document.getElementById('importInputModelType').value = 'keras';
+    }
+  } else if (onecc['one-import-tflite'] === 'True') {
+    checkboxImport.checked = true;
+    document.getElementById('importInputModelType').value = 'tflite';
+  } else if (onecc['one-import-onnx'] === 'True') {
+    checkboxImport.checked = true;
+    document.getElementById('importInputModelType').value = 'onnx';
+  } else if (onecc['one-import-bcq'] === 'True') {
+    checkboxImport.checked = true;
+    // TODO Enable when one-import-bcq is supported
+    // document.getElementById('importInputModelType').value = 'bcq';
+  }
+  checkboxOptimize.checked = onecc['one-optimize'] === 'True';
+  checkboxQuantize.checked = onecc['one-quantize'] === 'True';
+  checkboxCodegen.checked = onecc['one-codegen'] === 'True';
+  checkboxProfile.checked = onecc['one-profile'] === 'True';
+
+  // one-import Section
+  if (cfg['one-import-tf']) {
+    const oneImportTF = cfg['one-import-tf'];
+    const modelFormat = oneImportTF ?.['model_format'];
+    if (modelFormat === undefined || modelFormat === 'graph_def') {
+      document.getElementById('PBInputPath').value = cfgString(oneImportTF['input_path']);
+      document.getElementById('PBOutputPath').value = cfgString(oneImportTF['output_path']);
+      document.getElementById('PBConverterVersion').value =
+          cfgString(oneImportTF['converter_version']);
+      document.getElementById('PBInputArrays').value = cfgString(oneImportTF['input_arrays']);
+      document.getElementById('PBOutputArrays').value = cfgString(oneImportTF['output_arrays']);
+      document.getElementById('PBInputShapes').value = cfgString(oneImportTF['input_shapes']);
```

Comment:
@seanshpark mentioned that these are needed only when a model is from TF v1.

I wonder what would happen if there is no `input_arrays` key in cfg in case of TF v2 model.
In such case we'd better not show `PBInputArrays`, I believe.

Maybe we need some `TODO` here to support such situation.

Reply:
FYI, my explanation about `input_arrays` not needed in `V1` is incorrect. it was `graph_def` and current code looks OK.

Reply:
@hyunsik-yoon FYI :)

- `input_arrays` : Mandatory for both v1 and v2
- `output_arrays` : Mandatory for both v1 and v2
- `input_shapes` : Optional for both v1 and v2

Reply:
Please go on without my approval.